src/scene.ts

```diff
@@ -4300,24 +4300,22 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
 
         this._frameId++;
         this._defaultFrameBufferCleared = false;
-        if (this._activeCamera?.outputRenderTarget && !this.activeCamera?.isRigCamera) {
-            this._activeCamera.outputRenderTarget._cleared = false;
-        }
-        if (this._activeCamera?.rigCameras?.length) {
-            for (let i = 0; i < this._activeCamera.rigCameras.length; ++i) {
-                const rtt = this._activeCamera.rigCameras[i].outputRenderTarget;
-                if (rtt) {
-                    rtt._cleared = false;
-                }
+        const checkRtt = (camera: Nullable<Camera>) => {

```

Comment:
This would create GC is it possible to move in it s own function instead ?

Reply:
sure!